corehq/apps/app_manager/xform.py

```diff
@@ -954,7 +954,7 @@ def get_questions(self, langs, include_triggers=False,
         control_nodes = self.get_control_nodes()
         leaf_data_nodes = self.get_leaf_data_nodes()
 
-        for node, path, repeat, group, items, is_leaf, data_type, relevant, required in control_nodes:
+        for node, path, repeat, group, items, is_leaf, data_type, relevant, required, constraint in control_nodes:
```

Comment:
Yeah, it's a bit much. Switched to a namedtuple, which is at least a little bit more readable.